encode_test.go

```diff
@@ -47,7 +47,7 @@ func javaDecodeValidate(method string, target interface{}) (string, error) {
 
 	stdin, err := cmd.StdinPipe()
 	if err != nil {
-		fmt.Printf("call java error: %v", err)
+		fmt.Printf("call java error: %v\n", err)
```

Comment:
u should use t.Logf in go test

Reply:
there is not a testing.T parameter for the method

Reply:
this is not a right excuse. this func is invoked by testJavaDecode. so i think u can add a testing.T as this func's parameter.

Reply:
yes, u r correct